### PR TITLE
INJIMOB-2023 fixed inji-wallet ui automation for Ios and android

### PR DESCRIPTION
**Pull request by anup-nehe**
INJIMOB-2023 fixed inji-wallet ui automation for Ios and android

### CODE REVIEW

**Review by swatigoel — APPROVED** [review]

File: injitest/iosRegression.xml [hunk 1]
```diff
@@ -77,7 +75,6 @@
                     <include name="downloadAndVerifyVcUsingUinViaEsignet"/>
                     <include name="downloadAndVerifyVcUsingVidViaEsignet"/>
                     <include name="denyPopupforEsignetLoginWebPage"/>
-<!--                    <include name="downloadMultipleVcViaEsignet"/>  invalid testcase -->
```
Comment (sree96) [inline injitest/iosRegression.xml:80]: Remove unused code Please

File: injitest/src/test/java/iosTestCases/VerifyWelcomePagesTest.java [hunk 2]
```diff
@@ -185,4 +186,241 @@ public void verifyClickOnBackFromInjiTourGuide() {
...
+        assertTrue(homePage.isHomePageLoaded(), "Verify if home page is displayed");
+
+        AddNewCardPage addNewCardPage = homePage.downloadCard();
+        EsignetLoginPage esignetLoginPage =  addNewCardPage.clickOnDownloadViaEsignet();
+
+        esignetLoginPage.clickOnEsignetLoginWithOtpButton();
+        addNewCardPage.clickOnContinueButtonInSigninPopupIos();
+
+
+//        assertTrue(esignetLoginPage.isEnterYourVidTextDisplayed(), "Verify if enter your vid text is displayed");
```
Comment (sree96) [inline injitest/src/test/java/iosTestCases/VerifyWelcomePagesTest.java:219]: Remove unused code Please

File: injitest/src/test/java/iosTestCases/VerifyWelcomePagesTest.java [hunk 3]
```diff
@@ -185,4 +186,241 @@ public void verifyClickOnBackFromInjiTourGuide() {
...
+        assertTrue(addNewCardPage.isIssuerSearchBarDisplayed(), "Verify if issuer search bar displayed");
+        assertTrue(addNewCardPage.isAddNewCardPageLoaded(), "Verify if add new card page is displayed");
+        assertTrue(addNewCardPage.isAddNewCardPageGuideMessageForEsignetDisplayed(), "Verify if add new card guide message displayed");
+        assertTrue(addNewCardPage.isDownloadViaEsignetDisplayed(), "Verify if download via uin displayed");
+        EsignetLoginPage esignetLoginPage =  addNewCardPage.clickOnDownloadViaEsignet();
+
+        esignetLoginPage.clickOnEsignetLoginWithOtpButton();
+        addNewCardPage.clickOnContinueButtonInSigninPopupIos();
+
+//        assertTrue(esignetLoginPage.isEnterYourVidTextDisplayed(), "Verify if enter your vid text is displayed");
```
Comment (sree96) [inline injitest/src/test/java/iosTestCases/VerifyWelcomePagesTest.java:341]: Remove unused code Please

File: injitest/src/test/java/iosTestCases/VerifyWelcomePagesTest.java [hunk 4]
```diff
@@ -185,4 +186,241 @@ public void verifyClickOnBackFromInjiTourGuide() {
...
+        String uin=TestDataReader.readData("uin");
+        OtpVerificationPage otpVerification= esignetLoginPage.setEnterIdTextBox(uin);
+
+        esignetLoginPage.clickOnGetOtpButton();
+//        assertTrue(esignetLoginPage.isOtpHasSendMessageDisplayed(),"verify if otp page is displayed");
```
Comment (sree96) [inline injitest/src/test/java/iosTestCases/VerifyWelcomePagesTest.java:346]: Remove unused code Please

**Review by sree96 — CHANGES_REQUESTED** [review]
Please address the comments

**Review by sree96 — APPROVED** [review]

**Pull request merged**